packages/node_modules/@ciscospark/plugin-avatar/src/avatar-url-store.js

```diff
@@ -78,14 +78,13 @@ export default class AvatarUrlStore {
     if (!item.url) {
       return Promise.reject(new Error(`\`item.url\` is required`));
     }
-    if (!item.cacheControl) {
-      return Promise.reject(new Error(`\`item.cacheControl\` is required`));
+    if (!item.maxAge) {
+      return Promise.reject(new Error(`\`item.maxAge\` is required`));
     }
 
-    // use item suggested cache TTL or configed default if missing
-    setTimeout(this.remove.bind(this, item), item.cacheControl * 1000);
+    setTimeout(this.remove.bind(this, item), item.maxAge * 1000);
     urlByUuid.get(this).set(`${item.uuid} - ${item.size}`, item);
-    return Promise.resolve(item);
+    return Promise.resolve(item.url);
```

Comment:
It's kinda odd to put an object into a store, then only get one of its properties back as a result of the add. Can you move the item.url selection to the invocation site?